google-cloud-bigquery/lib/google/cloud/bigquery/project.rb

```diff
@@ -1372,6 +1373,7 @@ def external url, format: nil
         # Retrieves an existing dataset by ID.
         #
         # @param [String] dataset_id The ID of a dataset.
+        # @param [String] project_id The GCP Project where the dataset lives.
```

Comment:
Should this param come in order of appearance in the method signature, after `skip_lookup`?